tasks/postgres.yml

```diff
@@ -33,18 +32,34 @@
   when: ansible_os_family == "RedHat" and
         db.use_rds == false
 
+- name: install postgres server on Debian
+  apt:
+   name: 'postgresql-{{ dataverse_pg_version_short }}'
+   state: latest
+  when: ansible_os_family == "Debian" and
+        db.use_rds == false
+
+
 - name: install postgres client on RedHat / CentOS for RDS
   yum:
     name: 'postgresql{{ dataverse_pg_version_short }}'
     state: latest
   when: ansible_os_family == "RedHat" and
         db.use_rds == true
 
+- name: install postgres client on Debian
+  apt:
+    name: 'postgresql-client-{{ dataverse_pg_version_short }}'
+    state: latest
+  when: ansible_os_family == "Debian" and
+        db.use_rds == true
+
+
 - name: set postgresql init command
   set_fact:
     postgres_init: 'creates=/var/lib/pgsql/{{ db.postgres.version }}/data/postgresql.conf /usr/pgsql-{{ db.postgres.version }}/bin/postgresql{{ dataverse_pg_version_short }}-setup initdb'
-  when: ansible_os_family == "RedHat" and
-        db.use_rds == false
+    when: ansible_os_family == "RedHat" and
```

Comment:
won't this spacing break the conditional check?

Reply:
Yes, it will. Fixed.